src/parser/parse_op.h

```diff
@@ -91,7 +89,7 @@ inline std::ostream& operator<<(std::ostream& os, const ParseOp& p)
   {
     out << " :op " << p.d_op;
   }
-  if (p.d_kind != kind::NULL_EXPR)
+  if (p.d_kind != api::NULL_EXPR)
```

Comment:
Can we move the implementation of this function to the cpp file?

Reply:
Sure